service/src/main/java/uk/gov/hmcts/reform/fpl/service/DocmosisDocumentGeneratorService.java

```diff
@@ -41,39 +49,10 @@ public DocmosisDocument generateDocmosisDocument(Map<String, String> templateDat
         try {
             response = restTemplate.exchange(tornadoUrl, HttpMethod.POST, request, byte[].class).getBody();
         } catch (HttpClientErrorException.BadRequest ex) {
-            throw new RuntimeException("Docmosis document generation failed" + ex.getResponseBodyAsString());
+            logger.error("Docmosis document generation failed");
```

Comment:
👎 We need to log the responseBodyAsString otherwise we wouldn't now why it failed.